oak-jcr/src/test/resources/logback-test.xml

```diff
@@ -29,15 +31,19 @@
         </encoder>
     </appender>
 
+<!-- for debugging URLBinaryIT -->
+<!--
+    <logger name="com.amazonaws.http" level="DEBUG"/>
+    <logger name="sun.net.www.protocol.http.HttpURLConnection" level="DEBUG"/>
+-->
+
     <logger name="org.apache.jackrabbit.test.api" level="DEBUG"/>
     <logger name="com.mongodb.tcp" level="ERROR"/>
     <logger name="org.apache.jackrabbit.oak.jcr.operations" level="INFO"/>
     <logger name="org.apache.jackrabbit.oak.security.authentication" level="WARN"/>
 
     <root level="INFO">
-        <!--
```

Comment:
We usually don't log to the console.

Reply:
Yes, see my comment above, happy to just remove this entire file change from the patch.

This was for working with the unit tests in the IDE, so the log output is visible live in the IDE test runner, aligned with the test methods, without having to open a separate file.

IMO it makes sense to have the tests log to the console, but then make the choice in maven surefire setup to push the test log output to a file, this way developers in the IDE and mvn build output users are happy.